### DISCUSSION TITLE
Local Folder location

### DISCUSSION THREAD

**Discussion by steenblikrs** [beakerbrowser/beaker#1733] [General]
I just started using the "My Photo Album" Template and I would like to edit the photos outside of Beaker, like in Photoshop in order to batch resize them, and mass rename them. Where can I find the local folder location for my Hyperdrive on my computer?

I also wonder what dictates the order the photos show up in the final site?

hyper://85db788f9a281b94a2c04422cc8ecb1bd2375bfa623431a70b867b8a9222c3d6/

**Comment by pfrazee** [comment]
We're working on a "local folder sync" tool to allow this. When the next beta patch comes out (prerelease.5) give it a shot; the current one is still a little hard to work with.
